### PR TITLE
Adding walkthrough instructions for 1.7

### PR DESCRIPTION
**Pull request by arschles**
This is a follow-up to https://github.com/kubernetes-incubator/service-catalog/pull/1163

### CODE REVIEW

File: docs/walkthrough-1.7.md [hunk 1]
```diff
@@ -3,6 +3,323 @@
...
+helm install charts/ups-broker --name ups-broker --namespace ups-broker
+```
+
+# Step 2 - Creating a `ServiceBroker` Resource
+
+Because we haven't created any resources in the service-catalog API server yet,
+`kubectl get` will return an empty list of resources.
+
+```console
+kubectl get brokers,serviceclasses,instances,bindings
```
Comment (kibbles-n-bytes) [inline docs/walkthrough-1.7.md]: `brokers`, `instances`, `bindings` -> `servicebrokers`, `serviceinstances`, `servicebindings`

File: docs/walkthrough-1.7.md [hunk 2]
```diff
@@ -3,6 +3,323 @@
...
+No resources found
+```
+
+We'll register a broker server with the catalog by creating a new
+[`ServiceBroker`](../contrib/examples/walkthrough/ups-broker.yaml) resource.
+Do so with the following command:
+
+```console
+kubectl --context=service-catalog create -f contrib/examples/walkthrough/ups-broker.yaml
```
Comment (kibbles-n-bytes) [inline docs/walkthrough-1.7.md]: We don't need the context anymore here I think.

File: docs/walkthrough-1.7.md [hunk 3]
```diff
@@ -3,6 +3,323 @@
...
+```
+
+The output of that command should be the following:
+
+```console
+broker "ups-broker" created
```
Comment (kibbles-n-bytes) [inline docs/walkthrough-1.7.md]: resource rename

File: docs/walkthrough-1.7.md [hunk 4]
```diff
@@ -3,6 +3,323 @@
...
+```
+
+When we create this `ServiceBroker` resource, the service catalog controller responds
+by querying the broker server to see what services it offers and creates a
+`ServiceClass` for each.
+
+We can check the status of the broker using `kubectl get`:
+
+```console
+kubectl get brokers ups-broker -o yaml
```
Comment (kibbles-n-bytes) [inline docs/walkthrough-1.7.md]: resource rename

File: docs/walkthrough-1.7.md [hunk 5]
```diff
@@ -3,6 +3,323 @@
...
+```console
+instance "ups-instance" created
+```
+
+After the `ServiceInstance` is created, the service catalog controller will 
+communicate with the appropriate broker server to initiate provisioning. 
+Check the status of that process with this command:
+
+```console
+kubectl get instances -n test-ns ups-instance -o yaml
```
Comment (kibbles-n-bytes) [inline docs/walkthrough-1.7.md]: resource rename

File: docs/walkthrough-1.7.md [hunk 6]
```diff
@@ -3,6 +3,323 @@
...
+
+Notice that a new `Secret` named `ups-instance-credential` has been created.
+
+# Step 6 - Deleting the `ServiceInstanceCredential`
+
+Now, let's unbind from the instance. To do this, we simply *delete* the
+`ServiceInstanceCredential` resource that we previously created:
+
+```console
+kubectl delete -n test-ns bindings ups-instance-credential
```
Comment (kibbles-n-bytes) [inline docs/walkthrough-1.7.md]: resource rename

File: docs/walkthrough-1.7.md [hunk 7]
```diff
@@ -3,6 +3,323 @@
...
+default-token-3k61z   kubernetes.io/service-account-token   3         30m
+```
+
+# Step 7 - Deleting the `ServiceInstance`
+
+Now, we can deprovision the instance. To do this, we simply *delete* the
+`ServiceInstance` resource that we previously created:
+
+```console
+kubectl delete -n test-ns instances ups-instance
```
Comment (kibbles-n-bytes) [inline docs/walkthrough-1.7.md]: resource rename

File: docs/walkthrough-1.7.md [hunk 8]
```diff
@@ -3,6 +3,323 @@
...
+```
+
+# Step 8 - Deleting the `ServiceBroker`
+
+Next, we should remove the `ServiceBroker` resource. This tells the service
+catalog to remove the broker's services from the catalog. Do so with this
+command:
+
+```console
+kubectl delete brokers ups-broker
```
Comment (kibbles-n-bytes) [inline docs/walkthrough-1.7.md]: resource rename

File: docs/walkthrough-1.7.md (hunk 2 above)
Comment (arschles) [reply docs/walkthrough-1.7.md]: good catch - thanks @kibbles-n-bytes

File: docs/walkthrough-1.7.md (hunk 1 above)
Comment (arschles) [reply docs/walkthrough-1.7.md]: good catch - thanks @kibbles-n-bytes

Comment (arschles) [reply docs/walkthrough-1.7.md]: done

File: docs/walkthrough-1.7.md (hunk 2 above)
Comment (arschles) [reply docs/walkthrough-1.7.md]: done

File: docs/walkthrough-1.7.md (hunk 3 above)
Comment (arschles) [reply docs/walkthrough-1.7.md]: done

File: docs/walkthrough-1.7.md (hunk 4 above)
Comment (arschles) [reply docs/walkthrough-1.7.md]: done

File: docs/walkthrough-1.7.md (hunk 5 above)
Comment (arschles) [reply docs/walkthrough-1.7.md]: done

File: docs/walkthrough-1.7.md (hunk 6 above)
Comment (arschles) [reply docs/walkthrough-1.7.md]: done

File: docs/walkthrough-1.7.md (hunk 7 above)
Comment (arschles) [reply docs/walkthrough-1.7.md]: done

File: docs/walkthrough-1.7.md (hunk 8 above)
Comment (arschles) [reply docs/walkthrough-1.7.md]: done

**Comment by arschles** [comment]
Thanks for catching all the incorrect resource names @kibbles-n-bytes - I addressed all your comments.

**Comment by kibbles-n-bytes** [comment]
No problem. LGTM!

File: docs/walkthrough-1.7.md [hunk 9]
```diff
@@ -3,6 +3,323 @@
...
-This document is a work in progress. Instructions for the self-guided demo
-will be similar to those in the [1.6 walkthrough](./walkthrough-1.6.md),
-but note that the commands in that document will not work as-is.
+All commands in this document assume that you're operating out of the root
+of this repository.
+
+# Step 1 - Installing the UPS Broker Server
+
+In order to effectively demonstrate the service catalog, we need a sample
+broker server. We've built a "dummy" server called the User Provided Service
```
[the remark this answers is not in the document]
Comment (arschles) [reply docs/walkthrough-1.7.md]: good catch, I modified the wording to only use "we" to indicate the person running through the demo

**Comment by arschles** [comment]
@pmorie I addressed your comment. Can you take another look?

**Pull request merged**